QModPluginEmulator/QModManager.QModPluginGenerator.csproj

```diff
@@ -107,8 +110,7 @@ xcopy "$(TargetDir)QModManager.OculusNewtonsoftRedirect.dll" "$(SolutionDir)Vort
 xcopy "$(TargetDir)QModInstaller.dll" "$(SolutionDir)VortexBuild\$(Configuration)\BepInEx\plugins\QModManager\"  /I /Q /Y
 xcopy "$(TargetDir)QModInstaller.xml" "$(SolutionDir)VortexBuild\$(Configuration)\BepInEx\plugins\QModManager\"   /I /Q /Y
 
-powershell Compress-Archive -Path '$(SolutionDir)VortexBuild\$(Configuration)\BepInEx' -DestinationPath '$(SolutionDir)VortexBuild\QModManager_$(Configuration).zip' -Force
-powershell Compress-Archive -LiteralPath '$(SolutionDir)VortexBuild\$(Configuration)\doorstop_config.ini', '$(SolutionDir)VortexBuild\$(Configuration)\winhttp.dll' -DestinationPath '$(SolutionDir)VortexBuild\QModManager_$(Configuration).zip' -Update
+$(SolutionDir)packages\7-Zip.CommandLine.18.1.0\tools\7za.exe a "$(SolutionDir)VortexBuild\QModManager_$(Configuration).zip" "$(SolutionDir)VortexBuild\$(Configuration)\*"
```

Comment:
I usually use separate .bat file if the build event is more than a few lines. It can be added to the project and edited the same way as sources, with syntax highlighting and without blocking everything.

Reply:
Feel free.

Reply:
What happened with "wherever we touch the codebase" and all that? :)

Reply:
I didn't say no, I said feel free to make the change.

Reply:
Btw, I didn't write the build event, I am improving it with my modification. If you would like to *further* improve it with a .bat, you are free to do so, nobody is stopping you. I haven't had the time.